### ISSUE TITLE
Add custom fonts on exported HTML header

### ISSUE THREAD

**Issue by adrianmarraco** [artf/grapesjs-mjml#207]
Hi!

First of all I wanted to congratulate you for the incredible work developed in this GrapesJS project.

For one of the projects I am currently working on, we need to be able to include in the exported HTML the custom fonts that we had **included and used** on the editor.

I have seen that the method used to generate the HTML code is the `mjml2html` of the `mjmlio/mjml` library (`grapesjs-mjml-master\src\components\utils.js`). One of the parameters that this method accepts in its `options` is the `fonts` parameter:

```
const htmlOutput = mjml2html(`
  <mjml>
    <mj-body>
      <mj-section>
        <mj-column>
          <mj-text>
            Hello World!
          </mj-text>
        </mj-column>
      </mj-section>
    </mj-body>
  </mjml>
`, {
  fonts: {
    'Montserrat': 'https://fonts.googleapis.com/css?family=Montserrat:300,400,500,700',
    'Roboto': 'https://fonts.googleapis.com/css?family=Roboto:300,400,500,700'
  }
})
```

I've downloaded your release `v0.2.10` source code and I've made some changes locally:

- Added a new plugin option called `fonts` (`grapesjs-mjml-master\src\index.js`): an object (empty by default) with the same format of the code above. 
- Added a new parameter `fonts` on your method `mjmlConvert` (`grapesjs-mjml-master\src\components\utils.js`) and included on the method `mjml2html` call:

```
export function mjmlConvert (mjml, fonts) {
  return mjml2html(mjml, {
    useMjmlConfigOptions: false,
    mjmlConfigPath: null,
    filePath: null,
    fonts: fonts
  });
} 
```
- Added on every `mjmlConvert` call the plugin options parameter `fonts`:
  - `grapesjs-mjml-master\src\command-export-mjml.js` line 19
  - `grapesjs-mjml-master\src\components\Column.js` line 33
  - `grapesjs-mjml-master\src\components\Font.js` line 43
  - `grapesjs-mjml-master\src\components\index.js` line 181
  - `grapesjs-mjml-master\src\components\NavBar.js` line 51
  - `grapesjs-mjml-master\src\components\Style.js` line 35

The result of this is that in the exported HTML `<head>` appears the import of the fonts selected by the user:

```
<html>
  <head>
   ...
    <!--[if !mso]><!-->
    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;469;500;600;700&display=swap" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet" type="text/css">
    <style type="text/css">
      @import url(https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap);
      @import url(https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap);
      @import url(https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;469;500;600;700&display=swap);
      @import url(https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
      @import url(https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap);
    </style>
    <!--<![endif]-->
    ...
  </head>
  <body>
    ...
  </body>
</html>
```
 
Maybe is not the best way for add this functionality, but please take this as a suggestion so you can add it in future releases.

Thanks!

**Comment by DRoet** [comment]
If you are willing to PR a way to add fonts I would be OK with merging it

**Comment by adrianmarraco** [comment]
Hi @DRoet !

Is the first time for me in Github for propose a solution. I've clone the project repository locally and made this changes, but now I don't know how merge it.

**Comment by DRoet** [comment]
1. Make sure you fork the repository
2. Apply the changes and commit/push them to your fork
3. Open a PR in Github, make sure you click "compare across forks" and select the `master` branch as base branch

**Comment by adrianmarraco** [comment]
Thanks! I'll try right now!

**Comment by adrianmarraco** [comment]
Pull Request https://github.com/artf/grapesjs-mjml/pull/208

**Comment by DRoet** [comment]
I'll take a look later this week, I'll keep the issue open until it is merged

**Comment by adrianmarraco** [comment]
@DRoet OK Thanks!

**Comment by DRoet** [comment]
Released in `v0.3.0`
